example.dev-config.yml

```diff
@@ -53,4 +53,5 @@ pulp_settings:
   content_origin: "{{ pulp_webserver_disable_https | default(false) | ternary('http', 'https') }}://{{ ansible_fqdn }}"
   allowed_export_paths: ["/tmp"]
   allowed_import_paths: ["/tmp"]
+  use_new_tasking_system: true
```

Comment:
Is this the right name? https://github.com/pulp/pulpcore/blob/master/docs/configuration/settings.rst#use_new_worker_type

Reply:
```suggestion
  use_new_worker_type: true
```
Where's my brain?